src/content/releases/_index.en.md

```diff
@@ -5,6 +5,22 @@ pre = "<b>5. </b>"
 weight = 25
 +++
 
+## v0.6.0 Lighthouse improvements
+> This release is mainly focused in support for headless for Services and StatefulSets in Lighthouse,
+> and A/P HA enhancements and general stability for Submariner.
+
+* **Lighthouse** has been enhanced to:
+  * Be aware of the local cluster Gateway connectivity, and now does not announce
+    IP addresses or disconnected remote clusters.
+  * Support Headless services
+  * Stateful sets
+* **subctl** auto-detects the cluster-id from the kubeconfig file information when possible.
+* HA improvements for *Submariner*, the shutdown path of all agents has been analized, and
+  fixed in some cases, improving the shutdown when an agent needs to be moved to a different
+  node.
+* Minimum **K8s** version is now 1.17, as we begin support for EndpointSlices in Lighthouse
+* Multiple submariner bugfixes since 0.5.0
+
```

Comment:
* Hostnetwork to remoteService support fixes for AWS Cluster.

Notes: This is just for info. We fixed few edge cases where the hostnetwork to remoteService use-case was broken on AWS cluster. This is fixed via the following PRs.
Merged: https://github.com/submariner-io/submariner/pull/737
Merged: https://github.com/submariner-io/submariner/pull/738

Reply:
Done.